[2022-06-23 08:24:08.690] [remoteagent] [info] Extension host agent started.
[2022-06-23 08:24:09.293] [remoteagent] [error] [127.0.0.1][711ef096][ManagementConnection] Unknown reconnection token (never seen).
[2022-06-23 08:24:09.383] [remoteagent] [error] [127.0.0.1][93f39ebd][ExtensionHostConnection] Unknown reconnection token (never seen).
[2022-06-23 08:24:09.425] [remoteagent] [error] [/home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30]: Failed to parse /home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30/package.nls.json: Value expected.
[2022-06-23 08:24:09.427] [remoteagent] [error] [/home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30]: Failed to parse /home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30/package.nls.json: Value expected.
[2022-06-23 08:24:10.117] [remoteagent] [info] Deleted from disk ms-python.vscode-pylance /home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.20
[2022-06-23 08:24:10.138] [remoteagent] [error] [/home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30]: Failed to parse /home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30/package.nls.json: Value expected.
[2022-06-23 08:24:10.391] [remoteagent] [error] [/home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30]: Failed to parse /home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30/package.nls.json: Value expected.
[2022-06-23 08:24:15.922] [remoteagent] [info] [127.0.0.1][a2cfea15][ManagementConnection] New connection established.
[2022-06-23 08:24:15.996] [remoteagent] [info] [127.0.0.1][f8e740c0][ExtensionHostConnection] New connection established.
[2022-06-23 08:24:16.010] [remoteagent] [info] [127.0.0.1][f8e740c0][ExtensionHostConnection] <1362> Launched Extension Host Process.
[2022-06-23 08:24:16.435] [remoteagent] [error] [/home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30]: Failed to parse /home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30/package.nls.json: Value expected.
[2022-06-23 08:24:16.649] [remoteagent] [error] [/home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30]: Failed to parse /home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30/package.nls.json: Value expected.
[2022-06-23 08:24:17.055] [remoteagent] [error] [/home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30]: Failed to parse /home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30/package.nls.json: Value expected.
[2022-06-23 08:24:17.216] [remoteagent] [error] [/home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30]: Failed to parse /home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30/package.nls.json: Value expected.
[2022-06-23 08:24:17.394] [remoteagent] [error] [/home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30]: Failed to parse /home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30/package.nls.json: Value expected.
[2022-06-23 08:24:18.298] [remoteagent] [error] [/home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30]: Failed to parse /home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30/package.nls.json: Value expected.
[2022-06-23 08:24:18.459] [remoteagent] [error] ["Error: Could not find pty on pty host","    at w._throwIfNoPty (/home/pi/.vscode-server/bin/4af164ea3a06f701fe3e89a2bcbb421d2026b68f/out/vs/platform/terminal/node/ptyHostMain.js:20:5333)","    at w.updateTitle (/home/pi/.vscode-server/bin/4af164ea3a06f701fe3e89a2bcbb421d2026b68f/out/vs/platform/terminal/node/ptyHostMain.js:20:1673)","    at Object.call (/home/pi/.vscode-server/bin/4af164ea3a06f701fe3e89a2bcbb421d2026b68f/out/vs/platform/terminal/node/ptyHostMain.js:13:8386)","    at h.onPromise (/home/pi/.vscode-server/bin/4af164ea3a06f701fe3e89a2bcbb421d2026b68f/out/vs/platform/terminal/node/ptyHostMain.js:12:4891)","    at h.onRawMessage (/home/pi/.vscode-server/bin/4af164ea3a06f701fe3e89a2bcbb421d2026b68f/out/vs/platform/terminal/node/ptyHostMain.js:12:4274)","    at /home/pi/.vscode-server/bin/4af164ea3a06f701fe3e89a2bcbb421d2026b68f/out/vs/platform/terminal/node/ptyHostMain.js:12:3560","    at E.invoke (/home/pi/.vscode-server/bin/4af164ea3a06f701fe3e89a2bcbb421d2026b68f/out/vs/platform/terminal/node/ptyHostMain.js:8:145)","    at W.deliver (/home/pi/.vscode-server/bin/4af164ea3a06f701fe3e89a2bcbb421d2026b68f/out/vs/platform/terminal/node/ptyHostMain.js:8:2271)","    at c.fire (/home/pi/.vscode-server/bin/4af164ea3a06f701fe3e89a2bcbb421d2026b68f/out/vs/platform/terminal/node/ptyHostMain.js:8:1849)","    at process.U (/home/pi/.vscode-server/bin/4af164ea3a06f701fe3e89a2bcbb421d2026b68f/out/vs/platform/terminal/node/ptyHostMain.js:6:35698)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-06-23 08:24:20.078] [remoteagent] [error] [/home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30]: Failed to parse /home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30/package.nls.json: Value expected.
[2022-06-23 08:24:20.585] [remoteagent] [error] [/home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30]: Failed to parse /home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30/package.nls.json: Value expected.
[2022-06-23 08:29:19.880] [remoteagent] [error] [/home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30]: Failed to parse /home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30/package.nls.json: Value expected.
[2022-06-23 08:34:21.886] [remoteagent] [error] [/home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30]: Failed to parse /home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30/package.nls.json: Value expected.
[2022-06-23 08:39:22.024] [remoteagent] [error] [/home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30]: Failed to parse /home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30/package.nls.json: Value expected.
[2022-06-23 08:44:22.137] [remoteagent] [error] [/home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30]: Failed to parse /home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30/package.nls.json: Value expected.
[2022-06-23 08:49:22.269] [remoteagent] [error] [/home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30]: Failed to parse /home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30/package.nls.json: Value expected.
[2022-06-23 08:54:22.574] [remoteagent] [error] [/home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30]: Failed to parse /home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30/package.nls.json: Value expected.
[2022-06-23 08:59:22.709] [remoteagent] [error] [/home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30]: Failed to parse /home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30/package.nls.json: Value expected.
[2022-06-23 09:04:22.826] [remoteagent] [error] [/home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30]: Failed to parse /home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30/package.nls.json: Value expected.
[2022-06-23 09:09:23.047] [remoteagent] [error] [/home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30]: Failed to parse /home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30/package.nls.json: Value expected.
[2022-06-23 09:14:23.220] [remoteagent] [error] [/home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30]: Failed to parse /home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30/package.nls.json: Value expected.
[2022-06-23 09:19:23.398] [remoteagent] [error] [/home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30]: Failed to parse /home/pi/.vscode-server/extensions/ms-python.vscode-pylance-2022.6.30/package.nls.json: Value expected.
